components/airtable/new-modified-or-deleted-records.js

```diff
@@ -0,0 +1,111 @@
+const airtable = require('https://github.com/PipedreamHQ/pipedream/components/airtable/airtable.app.js');
+const moment = require('moment');
+const axios = require('axios');
+const Bottleneck = require('bottleneck');
+
+const limiter = new Bottleneck({
```

Comment:
Great use of rate-limiting functionality here

Reply:
Thank You.